packages/datastore/package.json

```
@@ -0,0 +1,43 @@
+{
+	"name": "@directus/datastore",
+	"version": "0.0.1",
+	"type": "module",
+	"scripts": {
+		"build": "tsc --build",
+		"dev": "tsc --watch",
+		"test": "vitest run",
+		"test:watch": "vitest",
+		"test:coverage": "vitest run --coverage"
+	},
+	"description": "Database layer for Directus.",
```

Sorry for nitpicking! 🙈 But I remember that we've removed the periods from the description in all packages some time ago... 😃 

```suggestion
	"description": "Database layer for Directus",
```

---

No problem, I didn't know :+1: